src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.md

```diff
@@ -363,6 +423,24 @@ UCS accepts these compaction strategy parameters:
   minimum compaction concurrency for the lowest levels. A low number would result in larger L0 sstables but may limit
   the overall maximum write throughput (as every piece of data has to go through L0).  
   The default value is 4 (1 for system tables, or when multiple data locations are defined).
```

Comment:
This default is now 4 for all tables; the description needs to mention that this only applies after `min_sstable_size` is reached.